api/controller/dbController.js

```diff
@@ -4,6 +4,18 @@ import {
   } from '../../db/queries'
 
   export class dbController {
+
+    songSearch = async ({name,artistId}) => {
+      const songEntryString = this.separateSongWords(name)
+      console.log(songEntryString,artistId)
+      return await query(songQueries.searchSong,[songEntryString,artistId])
+    }
+
+    separateSongWords = (entry) => {
+      const songArr = entry.split(' ')
+      const queryString = songArr.join('|')
+      return queryString
+    } 
```

Comment:
Rename this to be less specific, and _maybe_ move it out of the file? Only if you can find a good place to put it.